tests/v2/test_wakunode.nim

```diff
@@ -250,4 +250,37 @@ procSuite "WakuNode":
       node1Agent == expectedAgentString1
       node2Agent == expectedAgentString2
 
-    await allFutures(node1.stop(), node2.stop())
\ No newline at end of file
+    await allFutures(node1.stop(), node2.stop())
+
+  asyncTest "Custom multiaddresses are set and advertised correctly":
+    let
+      # custom multiaddress
+      expectedMultiaddress1 = MultiAddress.init("/ip4/200.200.200.200/tcp/1234").get()
+
+    # Note: this could have been done with a single node, but it is useful to
+    # have two nodes to check that the multiaddress is advertised correctly
+    let
+      # node with custom multiaddress
+      nodeKey1 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node1 = WakuNode.new(nodeKey1, ValidIpAddress.init("0.0.0.0"), Port(61018),
```

Comment:
im starting to use Port(0) which should get one among the available ones. just an fyi :)
https://github.com/waku-org/nwaku/blob/master/tests/v2/test_peer_manager.nim#L157

Reply:
Will address in a subsequent pr! Don't want to retrigger ci :)